ui/src/Groups/Counter/Counter.view.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Body, Subheading } from '../../Elements/Text/index.js';
+
+const Counter = (props) => {
+  const { title, subtitle, count, countLimit, className, ...others } = props;
+  const classesToApply = `Counter${className ? ` ${className}` : ''}`;
```

Comment:
https://www.npmjs.com/package/clsx - probably about time we bring something in (or roll our own utility)

Reply:
Added `clsx` & updated the Text component to use it too.